app/models/education_benefits_claim.rb

```diff
@@ -37,6 +37,10 @@ def parsed_form
     @parsed_form ||= JSON.parse(form)
   end
 
+  def confirmation_number
+    "vets_gov_#{self.class.to_s.underscore}_#{id}"
```

Comment:
There's an Admin Interface in VONAPP where admins can look up forms by submission ID, and print copies of them out. The call center just passes those confirmation numbers along to someone with access to that admin interface. We can use basically whatever we want, but we also may need to make some sort of 'retransmit' mechanism to send the 22-1990 to a different processing center.